connectors/sendgrid/src/test/java/io/camunda/connector/sendgrid/SendGridRequestTest.java

```diff
@@ -9,21 +9,17 @@
 import static org.assertj.core.api.Assertions.assertThat;
 import static org.junit.jupiter.api.Assertions.assertThrows;
 
-import io.camunda.connector.api.outbound.OutboundConnectorContext;
 import io.camunda.connector.impl.ConnectorInputException;
 import org.junit.jupiter.params.ParameterizedTest;
 import org.junit.jupiter.params.provider.MethodSource;
 
 public class SendGridRequestTest extends BaseTest {
 
-  private SendGridRequest sendGridRequest;
-  private OutboundConnectorContext context;
-
   @ParameterizedTest(name = "Validate null field # {index}")
   @MethodSource("failRequestCases")
   void validate_shouldThrowExceptionWhenLeastOneNotExistRequestField(String input) {
     // Given request without one required field
-    context = getContextBuilderWithSecrets().variables(input).build();
+    var context = getContextBuilderWithSecrets().variables(input).build();
     // When context.validate(sendGridRequest);
```

Comment:
nit: outdated comments (a few similar ones in this file)

Reply:
Ok, will take later to get this through.

Reply:
One more reason to avoid comments in these cases.